CONTRIBUTING.md

```diff
@@ -1,31 +1,31 @@
 
 ## Pull Request Process
 
-1. Ensure any install or build dependencies are removed before the end of the layer when doing a build
-2. Check that there are no conflicts and your request passes [Travis](https://travis-ci.org) build. Check the log of the pass test if it fails the build
-3. Give the description of the issue that you want to resolve in the pull request message. The format of the commit message to be fixed    - **Fixes #[issue number] [Description of the issue]** Example: **Fixes #529: Add toast warning in `MainActivity.java`**
-4. Wait for the maintainers to review your pull request and do the changes if requested
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Check that there are no conflicts and your request passes [Travis](https://travis-ci.org) build. Check the log of the pass test if it fails the build.
+3. Give the description of the issue that you want to resolve in the pull request message. The format of the commit message to be fixed is  - **Fixes #[issue number] [Description of the issue]** Example: **Fixes #529: Add toast warning in `MainActivity.java`**
+4. Wait for the maintainers to review your pull request and do the changes if requested.
 
 ## Contributions Best Practices
 
 ### Commits
 
-* Write clear meaningful git commit messages (Do read [here](https://chris.beams.io/posts/git-commit/))
-* Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. (For more info click [here](https://github.com/blog/1506-closing-issues-via-pull-requests))
-* When you make very very minor changes to a PR of yours (like for example fixing a failing Travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterward so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at [here](https://davidwalsh.name/squash-commits-git))
-* When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker
+* Write clear meaningful git commit messages (Do read [here](https://chris.beams.io/posts/git-commit/)).
+* Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged(For more info click [here](https://github.com/blog/1506-closing-issues-via-pull-requests)).
+* When you make very very minor changes to a PR of yours (like for example fixing a failing Travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterward so that you don't have an absurd number of commits for a very small fix(Learn how to squash at [here](https://davidwalsh.name/squash-commits-git)).
```

Comment:
```suggestion
* When you make very very minor changes to a PR of yours (like for example fixing a failing Travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterward so that you don't have an absurd number of commits for a very small fix (Learn how to squash at [here](https://davidwalsh.name/squash-commits-git)).
```